client/blocks/reader-featured-image/README.md

```diff
@@ -0,0 +1,17 @@
+# Reader Featured Image
+
+Displays a featured image for a Reader post card.
+
+## Example
+
+The `ReaderFeaturedImage` component can be used in much the same way that you would use an `<ExternalLink>` component. The link wraps whatever is placed between the ReaderFeaturedImage elements.
```

Comment:
I did mean `ExternalLink` because it works the same way, but you're right, it's probably not a helpful comparison here. I'll remove it 👍